Compile error

Resolve compile error
```
error[E0432]: unresolved import `crate::text_coloring`
 --> src/commands.rs:2:12
  |
2 | use crate::text_coloring::{toSuccess, toError, toPlainMessage, toWarn, toCyan};
  |            ^^^^^^^^^^^^^ could not find `text_coloring` in the crate root

```

--- a/src/main.rs
+++ b/src/main.rs
@@ -11,6 +11,7 @@ mod json_domain;
 mod smart_parser;
 mod logger;
 mod config_loader;
+mod text_coloring;
 
 use grpc_request_dsl::*;
 use user_input::*;

